doc/replication-policies.md

```diff
@@ -14,6 +14,27 @@ intervals.
 
 ## Android replication policies
 
+If your Android app is targeting Android 5.0 (API level 21) or above it is recommended that you use Android's [`JobScheduler`](https://developer.android.com/reference/android/app/job/JobScheduler.html)
+rather than the Replication Policies described in this section. Using the `JobScheduler` you can set various conditions
+for replication much like with Replication Policies and they make even more efficient use of device resources than Replication Policies.
+See the section [Replication using the JobScheduler](#replication-using-the-jobscheduler) for examples and advice regarding using
+the `JobScheduler`.
+
+If your app is targeting Android 7.0 (API level 24) or above, the `WifiPeriodicReplicationReceiver` will not work correctly because
+apps declaring a `BroadcastReceiver` in their manifest no longer receive the `android.net.conn.CONNECTIVITY_CHANGE` event. In
+this case, you should consider using `JobScheduler`.
```

Comment:
I think we should say `must` rather than `should consider` as it sounds like there isn't another choice.